app/src/main/java/swati4star/createpdf/activity/SplashActivity.java

```diff
@@ -1,10 +1,10 @@
 package swati4star.createpdf.activity;
 
 import android.content.Intent;
-import androidx.appcompat.app.AppCompatActivity;
```

Comment:
revert this file. No relevant change here

Reply:
Resolve in commit 48ac073